Beam/resources/assets/js/components/IotaHistogram.vue

```diff
@@ -0,0 +1,93 @@
+<style lang="scss">
```

Comment:
If we don't plan to use the styles globally, let's use `<style scoped...>` so they're prefixed by Vue and used only within this component and its children.

Reply:
Allright, I scoped all new components that needed scoping.